azure/storage/client.go

```diff
@@ -85,7 +88,7 @@ func NewClient(accountName, accountKey, blobServiceBaseUrl, apiVersion string, u
 		apiVersion:  apiVersion}, nil
 }
 
-func (c StorageClient) getBaseUrl(service string) string {
+func (c *StorageClient) getBaseUrl(service string) string {
```

Comment:
any particular reason making this a pointer receiver? none of the methods will actually change the `c` instance.